include/revng/BasicAnalyses/GeneratedCodeBasicInfo.h

```diff
@@ -56,6 +56,20 @@ class GeneratedCodeBasicInfo {
 
   void run(llvm::Module &M);
 
+  /// \brief Handle the invalidation of this information, so that it does not
+  /// get invalidated by other passes.
```

Comment:
Formal note: when you have `\brief` or `TODO:` we usually indent like this:
```suggestion
  /// \brief Handle the invalidation of this information, so that it does not
  ///        get invalidated by other passes.
```
Fix here and elsewhere.